test/unit/tests-api/actions-builder.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const q = require('q');
+
+const ActionsBuilder = require('../../../lib/tests-api/actions-builder');
+const util = require('../../util');
+
+describe('tests-api/actions-builder', () => {
+    const sandbox = sinon.sandbox.create();
+    const browser = util.makeBrowser();
+
+    const createActionsBuilder = (actions) => new ActionsBuilder(actions || []);
+
+    const runAction = (method, browser, postActions) => {
+        const state = util.makeStateStub();
+
+        createActionsBuilder(state.actions)[method]();
+        return state.actions[0](browser, postActions);
+    };
+
+    afterEach(() => sandbox.restore());
+
+    describe('changeOrientation', () => {
+        beforeEach(() => {
+            sandbox.stub(browser, 'getOrientation').returns(q());
+            sandbox.stub(browser, 'setOrientation').returns(q());
+        });
+
+        it('should throw in case of passed arguments', () => {
+            const fn = () => createActionsBuilder().changeOrientation('fake argument');
+
+            assert.throws(fn, TypeError, /\.changeOrientation\(\) does not accept any arguments/);
+        });
+
+        it('should return promise with ActionsBuilder instance', () => {
+            assert.instanceOf(createActionsBuilder().changeOrientation(), ActionsBuilder);
+        });
+
+        it('should change orientation from PORTRAIT to LANDSCAPE', () => {
+            browser.getOrientation.returns(q('PORTRAIT'));
+
+            return runAction('changeOrientation', browser)
+                .then(() => assert.calledWith(browser.setOrientation, 'LANDSCAPE'));
+        });
+
+        it('should change orientation from LANDSCAPE to PORTRAIT', () => {
+            browser.getOrientation.returns(q('LANDSCAPE'));
+
+            return runAction('changeOrientation', browser)
+                .then(() => assert.calledWith(browser.setOrientation, 'PORTRAIT'));
+        });
+
+        it('should change orientation back in post actions', () => {
```

Comment:
`should change orientation back in post actions` => `should restore orientation in post actions`